src/main/java/com/uber/cadence/internal/compatibility/proto/serviceclient/GrpcServiceStubs.java

```diff
@@ -91,7 +93,8 @@ final class GrpcServiceStubs implements IGrpcServiceStubs {
   private final MetaAPIGrpc.MetaAPIBlockingStub metaBlockingStub;
   private final MetaAPIGrpc.MetaAPIFutureStub metaFutureStub;
 
-  GrpcServiceStubs(ClientOptions options) {
+  @VisibleForTesting
+  public GrpcServiceStubs(ClientOptions options, boolean enableLogging) {
```

Comment:
instead of making it public, you could use the newInstance method https://github.com/uber/cadence-java-client/blob/bfd521dc895a59ffb2ba17875f828072e36a64b1/src/main/java/com/uber/cadence/internal/compatibility/proto/serviceclient/IGrpcServiceStubs.java#L38

Reply:
I initially made it public to pass through the enableLogging field without adding it to IGrpcServiceStubs#newInstance, but using the approach you've suggested I've enabled tracing and removed that field. Done.